src/libs/highlevel/README.md

```diff
@@ -166,32 +164,32 @@ The default callback simply logs the error with `ELEKTRA_LOG_DEBUG` and then cal
 
 TODO: why DEBUG? It should be ERROR? And we should enable logging by default, otherwise users will not see the message?
```

Comment:
The idea was, that most developers would provide their own fatal error handler, because they probably have their own log format and might need to do some cleanup before `exit()`. If we enable logging by default, we need to provide a way to disable it without using any low-level code.

Reply:
Then please document that it writing an fatal error handler is basically expected.